qemu/tests/cfg/multi_host.cfg

```diff
@@ -209,6 +209,77 @@
                       kdump_propagate_cmd = "kdumpctl propagate"
                       kdump_restart_cmd = "systemctl restart kdump.service"
                       kdump_status_cmd = "systemctl status kdump.service"
+                - with_xbzrle:
+                    type = migration_multi_host_with_xbzrle
+                    capabilitys = "xbzrle cache_size"
+                    capabilitys_state = "disable disable"
```

Comment:
Sorry, I wrote it wrong. I mean check the default state, and it is disabled. So I think no need to remove it.